css-flexbox/display-flex.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    div {
+      display: flex;
+      margin: 4px;
+    }
+
+    #card-deck {
+      padding: 2px;
+    }
+
+    .class {
```

Comment:
What is this selector for? I don't see any elements with `class="class"` (and thank god 😬 )